tests/test_api.py

```diff
@@ -458,6 +459,67 @@ def test_create_specification_auth(testclient):
     assert r.data.namespace.name == namespace
 
 
+# Only testing size values that will always cause errors. Smaller values could
+# cause errors as well, but would be flaky since the test conda-store state
+# directory might have different lengths on different systems, for instance,
+# due to different username lengths.
+@pytest.mark.parametrize(
+    "size",
+    [
+        # minio.error.MinioException: S3 operation failed; code:
+        # XMinioInvalidObjectName, message: Object name contains unsupported
+        # characters.
+        # The error message is misleading: it's a size issue.
```

Comment:
Yea this is concerning. Any idea why we get this error only with 255?

Reply:
It's a common limit in filesystems, for instance: https://serverfault.com/questions/9546/filename-length-limits-on-linux